src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/interview/InterviewSecondaryOutcomeMap.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ons.census.fwmt.outcomeservice.converter.interview;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class InterviewSecondaryOutcomeMap {
+
```

Comment:
create a private default constructor and get other classes to connect in a static way

Reply:
Still having issues with getting this to work as intended